# Interaction data (protein + complex)

## Protein

## Primary question: What are the key residues on the protein side?
## Sub-questions
1. Is there direct experimental data on the ssRNA-binding residues?
2. What are the functional sites on the protein? On which of them is the ssRNA expected to bind?
3. What is the predicted RNA interface?
3.a. What are the positively charged patches?
3.b. Are there exposed aromatics?
3.c. Are there conserved patches?

## Complex
1. Is there a protein-ssRNA complex, homologous to Nsp9, that we can use directly?
2. Is there direct experimental data on the contacts / interfaces?

On the nsp9 protein PDB 5HIZ, homologue of the nsp9 of cov19 (PDB 6W4B)
&quot;the nsp9-K10A and nsp9-R106A mutants have 1.3- and 2.7-fold reductions in ssDNA binding affinity compared to the wild-type protein, respectively, whereas the nsp9-K10A/R68A/K69A/R106A mutant shows much weaker binding affinity, with a 7.2-fold reduction, and nsp9-Y82A shows much stronger binding affinity, with an 8.0-fold increase&quot;
[https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096807/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096807/)

=> TODO: look if same residues in SARS-cov2 sequence
=> look if makes sense in the 5HIZ structure

nsp9 displays the same features that OB-fold proteins use to bind nucleic acids: a network of positively charged amino acids defines a positive track suitable for binding the phosphate backbone to the protein surface, whereas exposed aromatic residues might provide stacking interactions with nucleobases. These residues are conserved in all CoV nsp9 sequences (Arg-10, Lys-52, Trp-53, Arg-55, Arg-74, Phe-75, Lys-86, Tyr-87, Phe-90, Lys-92, Arg-99, and Arg-111), further suggesting that nsp9 is a nucleic acid-binding protein. In addition, two extended loops L23 and L45 display weak electron density associated with high _B_ factor values, indicating that they are flexible and/or mobile. They line the positively charged track, and they may clamp nucleic acids on the nsp9 surface after conformational change, as observed in other OB-fold proteins ([32](https://www.pnas.org/content/101/11/3792#ref-32)).
[https://www.pnas.org/content/101/11/3792](https://www.pnas.org/content/101/11/3792)

Surface plasmon resonance was used to demonstrate the function of nsp9 as a nucleic acid-binding protein. Biotinylated oligonucleotides bound to a streptavidin-coated solid support are able to bind nsp9 ([**Fig. 3**](https://www.pnas.org/content/101/11/3792#F3)[_ **A** _](https://www.pnas.org/content/101/11/3792#F3)). This function was confirmed by fluorescence experiments. As a fluorophore, nsp9 monomer has a single Trp residue (Trp-53), which is partially exposed to the solvent. The Trp-53 indole moiety is in a polar environment comprising side chains of Gln-20, Gly-66, and more remotely, Lys-52. Interactions of Trp-53 with ligand might therefore quench its fluorescence. This occurrence was indeed observed by using ssDNA and ssRNA oligonucleotides of defined sequence[2].

3. Is there other experimental data (e.g. SAXS) on the whole complex?

## Bibliography
[1] â€¦
[2] [https://www.pnas.org/content/101/11/3792](https://www.pnas.org/content/101/11/3792)
[]
